GroupB_TaskManagerProgram/task.py

```diff
@@ -1,18 +1,33 @@
 import datetime
 
 class Task:
-    def __init__(self, task_name, due_date):
+
+    #priority levels
+    PRIORITY_LOW = 0
+    PRIORITY_MEDIUM = 1
+    PRIORITY_HIGH = 2
+
+    # Completion statuses
+    STATUS_NOT_STARTED = "Not Started"
+    STATUS_IN_PROGRESS = "In progress"
+    STATUS_COMPLETED = "Completed"
+
+    def __init__(self, task_name, description, due_date, priority_level=PRIORITY_LOW, completion_status=0.0):

```

Comment:
i forgot to change the completion status!! it should be completion_status=NOT_STARTED